samples/MusicStore/ForTesting/Mocks/OpenIdConnect/OpenIdConnectBackChannelHttpHandler.cs

```diff
@@ -12,7 +13,12 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
             var response = new HttpResponseMessage();
 
             var basePath = Path.GetFullPath(Path.Combine(
-                Directory.GetCurrentDirectory(), "ForTesting", "Mocks", "OpenIdConnect"));
+#if NET451
```

Comment:
Is this just a bug? That seems quite odd, no? I would imagine that tons of tests would break during migration (not just ours, but anyone's tests).